hello-world/hello_world_test.rb

```diff
@@ -12,33 +12,25 @@
   exit 1
 end
 
+# Test data version:
+# 7668b09 Added hello world test definition
+
 class HelloWorldTest < Minitest::Test
   def test_no_name
-    assert_equal 'Hello, World!', HelloWorld.hello, <<-MSG
-    When given no name, it should greet the world!
-    MSG
+    assert_equal 'Hello, World!', HelloWorld.hello,
+                 'When given no name, it should greet the world!'
   end
```

Comment:
I really don't like it when a line is split onto a second line just to make it not exceed 80 characters. I know that it's common in Ruby, but it looks inconsistent, is much harder to scan, and seems like an exercise in box-ticking (_let's not let Rubocop complain, people!_).

If you don't like the HEREDOC's let's assign to a temporary variable above the assertion instead.